waterloop-site/src/pages/404.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "../theme/styles.scss";
+import Goose from "../static/img/404page/errorgoose.png"; 
+
+
+const ErrorPage: React.FC = () => (
+    <div className="errorpage">
+        <div className="errorpagetext">
+        <h3>ERROR</h3>
+        <h1>404</h1>
+        <p>Oops! The page you are looking for was not found.</p>
+        </div>
+        <div className="errorpageimage">
+        <img className="errorgoose" src={Goose}></img>
+        </div>
+    </div>
```

Comment:
We'll get into this next meeting, but we have a formatter for code in order for code to be in a consistent format, after next meeting apply the formatter to this (and to your css code as well).